features/user_can_register.feature

```diff
@@ -0,0 +1,41 @@
+Feature: User can register
+As a user
+In order to access my account
+I would like to have a signup/login function
```

Comment:
Login flow is already tested in another feature file. What does your PT story tell you and why?

Reply:
Removed login from the feature